pulp_2_tests/tests/rpm/api_v2/test_modular_errata.py

```diff
@@ -25,6 +25,7 @@
     get_repodata,
     get_xml_content_from_fixture,
 )
+from pulp_2_tests.tests.rpm.utils import check_issue_4405
```

Comment:
It does look like that all the tests present in this module are affected by the #4405.

What about we verify the #4405 just once? Something like...

```python
def setUpModule():  # pylint:disable=invalid-name
    """Possibly skip the tests in this module.

    Skip tests if `Pulp #4405 <https://pulp.plan.io/issues/4405>`_ affects us.
    """
    set_up_module()
    if check_issue_4405(config.get_config()):
        raise unittest.SkipTest('https://pulp.plan.io/issues/4405')

```

Reply:
@kersommoura yup, that is accurate. I hadn't done a `setupModule()` before, so I pushed the way it was looking for feedback.

Updating, testing and re-pushing soon. Thanks!

Reply:
This is something new to me as well. Thanks @kersommoura .

Reply:
@kersommoura updated test_modularity,py, as it had multiple classes and `test_modular_errata.py` only had a single instance of `check_issue`, per out bluejeans conversation.

Tested, make all, and re-pushed with updates.

Enjoy!